### ISSUE TITLE
Transparency does not work whatsoever

### ISSUE THREAD

**Issue by McNinjaTNT** [bonzibudd/Fluent-Refresh#11]
Menu transparency does not work in Windows 11. This goes for the original Fluent-Metro skin as well. Seems to be an issue with open-shell/W11 compatibility, as the skin just uses alpha channel in bitmaps.

Windows version tested: 22621.674

Figured I'd bring it up here as well, even though it's more of an open-shell issue. (See issue [#773](https://github.com/Open-Shell/Open-Shell-Menu/issues/773))

**Comment by McNinjaTNT** [comment]
Narrowed it down. Has to do with the "Enable menu glass" setting. When disabled, transparency works as expected. However shadows don't work right without this setting enabled.

**Comment by bonzibudd** [comment]
Yes, this has to do with the fact that Aero Glass has been removed from Windows 11. The normal transparency has the glitch with the shadow because it doesn’t map to the fully transparent section of the background. I’m not planning a fix, because I’d rather it be resolved on Open-Shell’s end, either making it’s own transparency engine or adapting to the Acrylic element. I don’t have the coding knowledge to figure that one out.
